src/useRoom.ts

```diff
@@ -68,27 +68,31 @@ export function useRoom(options?: RoomOptions): RoomState {
           });
           setAudioTracks(tracks);
         };
-
         newRoom.once(RoomEvent.Disconnected, () => {
           setTimeout(() => setRoom(undefined));
 
-          newRoom.off(RoomEvent.ParticipantConnected, onParticipantsChanged);
-          newRoom.off(RoomEvent.ParticipantDisconnected, onParticipantsChanged);
-          newRoom.off(RoomEvent.ActiveSpeakersChanged, onParticipantsChanged);
-          newRoom.off(RoomEvent.TrackSubscribed, onSubscribedTrackChanged);
-          newRoom.off(RoomEvent.TrackUnsubscribed, onSubscribedTrackChanged);
-          newRoom.off(RoomEvent.LocalTrackPublished, onSubscribedTrackChanged);
+          newRoom
+            .off(RoomEvent.ParticipantConnected, onParticipantsChanged)
+            .off(RoomEvent.ParticipantDisconnected, onParticipantsChanged)
+            .off(RoomEvent.ActiveSpeakersChanged, onParticipantsChanged)
+            .off(RoomEvent.TrackSubscribed, onSubscribedTrackChanged)
+            .off(RoomEvent.TrackUnsubscribed, onSubscribedTrackChanged)
+            .off(RoomEvent.LocalTrackPublished, onSubscribedTrackChanged)
+            // trigger a change by re-sorting participants
+            .off(RoomEvent.AudioPlaybackStatusChanged, onParticipantsChanged);
```

Comment:
I don't understand what this comment means

Reply:
ah the comment should've be left on the `.on`, but the idea is to trigger a refresh via a `setState` when it's possible to start audio playback.